DPG/Tasks/TPC/tpcSkimsTableCreator.cxx

```diff
@@ -0,0 +1,533 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file tpcSkimsTableCreator.cxx
+/// \brief Task to produce table with clean selections for TPC PID calibration
+///
+/// \author Annalena Kalteyer <annalena.sophie.kalteyer@cern.ch>
+/// \author Christian Sonnabend <christian.sonnabend@cern.ch>
+/// \author Jeremy Wilkinson <jeremy.wilkinson@cern.ch>
+/// O2
+#include <CCDB/BasicCCDBManager.h>
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/runDataProcessing.h"
+/// O2Physics
+#include "Common/Core/trackUtilities.h"
+#include "Common/DataModel/PIDResponse.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Common/DataModel/StrangenessTables.h"
+#include "Common/DataModel/Multiplicity.h"
+/// ROOT
+#include "TRandom3.h"
+#include <cmath>
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+using namespace o2::track;
+using namespace o2::dataformats;
+
+// void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
+// {
+//   std::vector<ConfigParamSpec> options{// runtime customisation goes here
+//                                        {"useV0", VariantType::Int, 0, {"Use V0 information for QA"}}};
+//   std::swap(workflowOptions, options);
+// }
+
+namespace o2::aod
+{
+namespace tpcskims
```

Comment:
Here I have a suggestion that you might consider, feel free to ignore it as well. If you move the definition of the table to a header file you could a)  clean the actual code b) use it later in some lite version of the task that runs of skimmed data. This is how we do in the DPG.